src/view.c

```diff
@@ -342,15 +354,11 @@ view_apply_region_geometry(struct view *view)
 {
 	assert(view);
 	assert(view->tiled_region || view->tiled_region_evacuate);
+	struct output *output = view->output;
+	assert(output_is_usable(output));
```

Comment:
Couldn't this assert trigger when there are not outputs left?

Reply:
The calling paths should prevent that now, see the new checks in `view_apply_special_geometry()` and `view_snap_to_region()`.